website/docs/guides/0.8.2-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,59 @@
+## Issue [INTMDB-175](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/398)
+### resources created with the affected versions ***v0.8.0/v0.8.1***
```

Comment:
This needs a better header to make it understandable, no one outside of MDB will understand the INTMDB number try:

## 0.8.2 Upgrade Guide for Privatelink users 

### Resources are impacted that were created with versions ***v0.8.0/v0.8.1***
### Fixed in [#398](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/398)